File: src/main/java/at/yawk/wm/wallpaper/animate/AnimatedWallpaperConfig.java
package at.yawk.wm.wallpaper.animate;

import java.awt.*;
import java.nio.file.Path;
import lombok.Data;

/**
 * @author yawkat
 */
@Data
public class AnimatedWallpaperConfig {
    private Path cache;
    private Path input;
    private Color backgroundColor;
}

File: src/main/java/at/yawk/wm/wallpaper/animate/AnimatedWallpaperConfig.kt
package at.yawk.wm.wallpaper.animate

import at.yawk.wm.style.Color
import java.nio.file.Path
import java.nio.file.Paths

/**
 * @author yawkat
 */
object AnimatedWallpaperConfig {
    val cache: Path = Paths.get(".cache/wallpaper")
    val input: Path = Paths.get("wallpaper")
    val backgroundColor = Color.Solarized.base03
    const val show = false
}
